File: jfm/src/tmpl/Template.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
package tmpl;

import java.io.File;
import java.io.FileNotFoundException;
import java.io.FileReader;
import java.io.IOException;
import javax.servlet.http.HttpServletRequest;
import javax.servlet.http.HttpServletResponse;
import net.Base;

/**
 *
 * @author anoop
 */
public class Template extends Base{

    public static void getTemplate(HttpServletRequest req, HttpServletResponse resp) throws FileNotFoundException, IOException {

        String userprofile = (String) System.getenv().get("USERPROFILE");
        File f = new File(userprofile + "/SkyDrive/workspace/StructJS1/importmanager.git/trunk/web/html/" + req.getParameter("data") + ".html");
        FileReader fr = new FileReader(f);
        String str = "";
        int c = fr.read();
        while (c != -1) {
            str += (char) c;
            c = fr.read();
        }
        fr.close();
        resp.getWriter().write(str);
        resp.getWriter().close();
    }
    @Override
    public void process(HttpServletRequest request, HttpServletResponse response) {
        throw new UnsupportedOperationException("Not supported yet.");
    }
}

File: jfm/src/node/test/Template.js
/* 
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */

var fs = require('fs');
fm.Package("test");
fm.Import("cookie.Cookie");
fm.Class("Template", "Base");
test.Template = function( base, me, Cookie,  Base){this.setMe=function(_me){me=_me;};

	this.getTemplate = function( req, resp, t ) {
//		if (!req.session && req.params.data != 'login') {
//			resp.writeHead(307, {
//				'Content-Type' : 'text/plain'
//			});
//			resp.write("login");
//			resp.end();
//			return;
//		}
		var path = process.cwd() + "/web/templates/" + req.params.data + ".html";
		console.log(path, "ddd"   );
		fs.readFile(path, function( err, data ) {
			if (err) {
				console.log(err);
			} 
			else {
				resp.write(data);
				resp.end("\n");
			}
			console.log(new Date().getTime() - t);
		});
		
	};
	this.method = function( ) {/**/};
};
console.log("Anoop ");






